src/pocketmine/resources/pocketmine.yml

```diff
@@ -121,6 +121,50 @@ chunk-ticking:
  tick-radius: 3
  light-updates: false
  clear-tick-list: true
+ 
+block-ticking:
+ #Configure which blocks should tick and which not
+ #Useful on slow PC's and to disable ice melting, fire burning down things etc.
+ #Grass
```

Comment:
Actually.. Just had a thought on that.. Maybe i was blind by coding xD
This way is not bad, :+1: for that. I'll rewrite later at home



> Am 05.01.2016 um 16:41 schrieb PEMapModder <notifications@github.com>:
> 
> In src/pocketmine/resources/pocketmine.yml:
> 
> > @@ -121,6 +121,50 @@ chunk-ticking:
> >   tick-radius: 3
> >   light-updates: false
> >   clear-tick-list: true
> > + 
> > +block-ticking:
> > + #Configure which blocks should tick and which not
> > + #Useful on slow PC's and to disable ice melting, fire burning down things etc.
> > + #Grass
> Why not simply let users use an array to write down what blocks to not tick?
> 
> disable-block-ticking: [2, 6, 59, 81]
> —
> Reply to this email directly or view it on GitHub.
>